beacon-light-client/plonky2/balance_verification/get_balances_input.ts

```diff
@@ -8,56 +8,51 @@ import { KeyPrefix, WorkQueue, Item } from '@mevitae/redis-work-queue';
 import validator_commitment_constants from '../constants/validator_commitment_constants.json';
 import { computeEpochAt } from '@dendreth/utils/ts-utils/ssz-utils';
 import { panic } from '@dendreth/utils/ts-utils/common-utils';
-import { CommandLineOptionsBuilder } from '../cmdline';
-import config from '../common_config.json';
+import commonConfig from '../common_config.json';
 import {
   convertValidatorToValidatorPoseidonInput,
   getZeroValidatorPoseidonInput,
 } from './utils';
 
```

Comment:
```suggestion
import {
  convertValidatorToValidatorPoseidonInput,
  getZeroValidatorPoseidonInput,
} from './utils';

import commonConfig from '../common_config.json';

interface CommonConfig {
  "redis-host": Address;
  "redis-port": Port;
  "beacon-node": string[];
}

const commonConfigChecked = commonConfig satisfies CommonConfig;
```

This way when you parse the JSON file you can validate it against this interface to ensure that it satisfies the expected structure. ( maybe this file is not the best place for the interface tho )